include/input.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "raylib.h"
+#include "module.hpp"
+#include <array>
+#include <vector>
+#include <string>
+#include <unordered_map>
+
+#define MAX_MODIFIERS 2
+
+typedef std::function<void()> ActionCallback;
+
+struct Action {
+    std::string name;
+    ActionCallback callback;
+};
+
+struct ActionShortcut {
+    KeyboardKey key;
+    std::array<KeyboardKey, MAX_MODIFIERS> modifiers;
+
+    bool shouldFire(int clickedKey);
+};
+
+struct InputMgr : ModuleSymbol {
+    std::vector<const Action> actions;
+    std::unordered_map<std::string, const ActionShortcut> shortcuts;
```

Comment:
std::vector nie bierze typów CV, jeżeli to kompiluje, to w niezgodzie ze standardem. Trzeba usunąć const.

Reply:
poprawione